### DISCUSSION TITLE
Introducing SQL DB plugins for Nextflow

### DISCUSSION THREAD

**Discussion by pditommaso** [nextflow-io/nf-sqldb#1] [Announcements]
This plugin allows Nextflow pipelines to query and populate SQL databases. The current version provides support for the most popular open-source databases such as MySQL and PostgreSQL. 

Find more details how to configure and use in the project [README](https://github.com/nextflow-io/nf-sqldb#readme) file. 

NOTE: This is a very early release. Configuration and features provided by this plugin can change in future versions. 

Please use this discussion area or the usual GitHub issue feature to leave your comment, suggestion, problem reporting or any other feedback!

**Comment by alanhoyle** [comment]
Oracle support would be great.  Our main back-end systems use Oracle for for their database, and it would be really convenient if we could query those directly.
